oauth/redirect/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Redirecting back to your app</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <style>
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+      body {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-family: Arial, sans-serif;
+        font-size: 14px;
+        line-height: 1.4;
+        text-align: center;
+        padding: 0 1rem;
+      }
+      h1 {
+        font-size: 20px;
+      }
+      p {
+        font-size: 14px;
+      }
+      p.error {
+        color: red;
+        font-weight: bold;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Redirecting back to your app...</h1>
+    <p>Please wait...</p>
+    <script>
+      (function () {
+        "use strict";
+
+        const displayError = (message) => {
+          document.body.innerHTML = `<p class="error">${message}</p>`;
```

Comment:
Same feedback here with the redirect back to originating application button/similar so the user isnt just stuck on this page